src/systemtest/resources/features/ArtifactFlow.feature

```diff
@@ -4,13 +4,53 @@ Feature: Artifact system test
   @ArtifactFlowScenario
   Scenario: Artifact Flow Scenario
     Given configurations are provided
-    And some subscriptions are set up
-    And a jenkins job '"ArtC2Job"' from '"src/systemtest/resources/JenkinsShellScripts/ArtC2Script.txt"' is created
+    When a jenkins job '"ArtC2Job"' from '"src/systemtest/resources/JenkinsShellScripts/ArtC2Script.txt"' is created
     And a jenkins job '"TCTJob"' from '"src/systemtest/resources/JenkinsShellScripts/TCTScript.txt"' is created
     And a jenkins job '"TCSTCFJob"' from '"src/systemtest/resources/JenkinsShellScripts/TCSTCFScript.txt"' is created
     And a jenkins job '"CLMJob"' from '"src/systemtest/resources/JenkinsShellScripts/CLMScript.txt"' is created
-    And a jenkins job '"ArtP"' from '"src/systemtest/resources/JenkinsShellScripts/ArtPScript.txt"' is created
-    When next story happens
-    Then all is good
+    And a jenkins job '"ArtPJob"' from '"src/systemtest/resources/JenkinsShellScripts/ArtPScript.txt"' is created
+    And a jenkins job '"FlowCompleteJob"' from '"src/systemtest/resources/JenkinsShellScripts/FlowCompleteScript.txt"' is created
+    Then we continue with the next step
+
+    #####Add CLMSubscription to EI#####
+    Given subscription "CLMSubscription" is created which will trigger "CLMJob"
```

Comment:
You create the subscription data, I was confused thinking that you sent it to EI at this step.